glide.lock

```diff
@@ -140,10 +151,6 @@ imports:
   - vfsutil
 - name: github.com/shurcooL/vfsgen
   version: 6a9ea43bcacdf716a5c1b38efff722c07adf0069
-- name: github.com/sirupsen/logrus
```

Comment:
we need this `github.com/sirupsen/logrus`
we don't need `github.com/Sirupsen/logrus`
the difference is the lowercase and upper case. Lowercase is required (this is an issue all go devs who use this library face because it did a weird rename between uppercase/lowercase)